workstation.yml

```diff
@@ -43,6 +43,33 @@
       until: "webconsole_uri is not failed"
       when: (cluster_state | default('present')) == 'present'
 
+    - name: Add instance to inventory
+      add_host:
+        name: "{{ cluster_name }}-{{ cluster_type }}"
+        groups: workstation
+        ansible_host: "{{ cluster_floating_ip_address }}"
+        ansible_user: ubuntu
+        ansible_ssh_private_key_file: "{{ cluster_ssh_private_key_file }}"
+      when:
+        - cluster_floating_ip_address is defined and cluster_floating_ip_address
+        - (cluster_state | default('present')) == 'present'
+
+- hosts: workstation # so only happens with FIP
+  gather_facts: no
+  become: yes
+  vars:
+    os_manila_mount_shares:
+      - share_name: azimuth-project-share
```

Comment:
I was wondering about using the extra var I am setting that has this value in, but lets not worry about that for now, as I hardcoded the name in Azimuth, rather than expose it via config, anyways.